src/Plugin.php

```diff
@@ -113,6 +126,12 @@ public function logDebug($message)
 
     public function handleIrcReceived(UserEvent $event, EventQueue $queue)
     {
+        if (
+            $this->filter !== null &&
+            $this->filter->filter($event) !== false
+        ) {
+            return;
+        }
```

Comment:
The original impetus for filing #11 was to facilitate filtering out URLs by hostname. If the event filter doesn't have access to the parsed URLs or has to parse them a second time, it seems to defeat that point. It's already possible to use this plugin with the EventFilter plugin via configuration without any changes to this plugin (see [usage documentation](https://github.com/phergie/phergie-irc-plugin-react-eventfilter#usage)). What I want is filtering specific to URLs.